plugins/statistics.koplugin/main.lua

```diff
@@ -1048,6 +1059,25 @@ function ReaderStatistics:getPageTimeTotalStats(id_book)
     return total_pages, total_time
 end
 
+function ReaderStatistics:usePageMapForPageNumbers()
+    if not self.ui.pagemap then
+        return false
+    elseif not self.ui.pagemap.has_pagemap then
+        return false
+    end
+
+    if self.ui.doc_settings:has("pagemap_use_page_labels")  then
+            if self.ui.doc_settings:isTrue("pagemap_use_page_labels") then
+                return true
+            end
+            return false
```

Comment:
```suggestion
        return self.ui.doc_settings:isTrue("pagemap_use_page_labels") 
```

Little indentation issue (8 spaces) in your original code - please check you're using everywhere 4 spaces by level.

(And the blank lines above and below can go :))